### ISSUE TITLE
BlueBlocker integration missing after new update

### ISSUE THREAD

**Issue by allstonecrop** [bethylamine/soupcan#125]
The "block users tagged from Soupcan" integration seems to have gone missing from BlueBlocker after a recent Soupcan update - I gained the star rating and lost the integration at the same time, I think.
I'm using Soupcan 0.22.0 and BlueBlocker v0.3.5. Please let me know if you need more info!

**Comment by INSANEMODE** [comment]
```06:47:49.205 Uncaught (in promise) ReferenceError: getDatabaseEntry is not defined
    <anonymous> moz-extension://e9905f74-9944-477d-8d30-ce8773972b21/background.js:222
    <anonymous> moz-extension://e9905f74-9944-477d-8d30-ce8773972b21/background.js:239
background.js:222:23
```

Seems to be a regression from #121, where getDatabaseEntry() got deleted from database.js, which background.js has as an imported script

**Comment by INSANEMODE** [comment]
tested adding  getDatabaseEntry() and hash() back to database.js, and the option appears again in blue blocker, so that is definitely the issue.

**Comment by bethylamine** [comment]
Oops! Didn't realize it needed that. Humm, okay.

**Comment by bethylamine** [comment]
Should be fixed in 0.22.2.

**Comment by bethylamine** [comment]
Btw, those two functions are duplicated (now). And I'm dumb, so I'm not sure how to unify them.
